File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.c
#include <stdio.h>

int main(void) {
    int students[30] = {0};
    int input = 0;
    
    for (int i = 0; i < 28; ++i) {
        scanf("%d", &input);
        students[input - 1] = input;
    }
    
    for (int j = 0; j < sizeof(students) / 4; ++j)
        if (students[j] != j + 1)
            printf("%d\n", j + 1);
}

File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int[] students = new int[30];
		
		for(int i = 0; i < 28; ++i) {
			int input = sc.nextInt();
			students[input - 1] = input;
		}
		
		for(int j = 0; j < 30; ++j) {
			if(students[j] != j + 1)
				System.out.println(j + 1);
		}
		
	}
}